pkg/kube/pod.go

```diff
@@ -235,9 +234,9 @@ func checkNodesStatus(p *v1.Pod, cli kubernetes.Interface) error {
 }
 
 // checkPVCAndPVStatus does the following:
-// 1. if PVC is present then check the status of PVC
-// 	1.1 if PVC is pending then check if the PV status is VolumeFailed return error if so. if not then wait for timeout.
-// 2. if PVC not present then wait for timeout
+// - if PVC is present then check the status of PVC
+// - if PVC is pending then check if the PV status is VolumeFailed return error if so. if not then wait for timeout.
+// - if PVC not present then wait for timeout
```

Comment:
tiny nit: [Go Doc Comments](https://go.dev/doc/comment#lists) says that bullet lists should be indented, so this should be:

```suggestion
//   - if PVC is present then check the status of PVC
//   - if PVC is pending then check if the PV status is VolumeFailed return error if so. if not then wait for timeout.
//   - if PVC not present then wait for timeout
```